### PR TITLE
feat: ✨  Added Gochain network (ws API)

### PR DESCRIPTION
**Pull request by r-gochain**
### Feature

* \[ ] Added entry to ./changelog
* \[ ] Is this a user submitted feature request?
  * \[ ] Link to issue:
* \[ ] Add test cases or procedure
* \[ ] Add PR label
* Adding Gochain websocket API endpoints.

### CODE REVIEW

File: src/utils/networks/types/GO.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  currencyName: 'GO',
+  isTestNetwork: false,
+  isEthVMSupported: {
+    supported: false,
+    url: null,
+    blockExplorerTX: '',
+    blockExplorerAddr: '',
+    websocket: null
+  },
+  gasPriceMultiplier: 1
```
Comment (kvhnuke) [inline src/utils/networks/types/GO.js:24]: can you please add the `coingeckID` I believe it is 'gochain' 
https://api.coingecko.com/api/v3/coins/list?include_platform=true

**Review by kvhnuke — CHANGES_REQUESTED** [review]

File: src/utils/networks/types/GO.js (hunk 1 above)
Comment (r-gochain) [reply src/utils/networks/types/GO.js:25]: Added (didn't find any other usages of coingeckID in the code)

**Review by rsteubs — APPROVED** [review]

**Comment by gamalielhere** [comment]
Added into #3442

**Comment by gamalielhere** [comment]
@r-gochain we're running into an issue with the node you submitted: "received a broken close frame containing a reserved status code.". Please take a look at it as we might have to remove gochain until this is resolved.

**Comment by treeder** [comment]
Hi, we'll look into this right away. Could you could leave it in for another day or so so we can debug?

**Comment by treeder** [comment]
@gamalielhere do you know how to reproduce the issue?  I just tried sending some GO using metamask via MyEtherWallet with no problem.

**Comment by r-gochain** [comment]
@gamalielhere could you please share the way to reproduce this problem? I just checked and I can connect using different wallets and wscat from the command line without any issues. Maybe some code to reproduce?

image

**Comment by gamalielhere** [comment]
Screen Shot 2022-01-18 at 2 55 46 PM

@r-gochain @treeder this is on MEW after accessing the wallet or any type of web3 calls with the gochain node it seems like

**Comment by r-gochain** [comment]
Thanks for sharing, I'll reproduce and try to fix.

**Comment by r-gochain** [comment]
@gamalielhere could you please check, we've updated the proxy and now it doesn't throw such errors

**Comment by gamalielhere** [comment]
@r-gochain seems like the error is gone now, but for some reason it's still not resolving the response to web3. We're also looking into it on our end.

**Comment by r-gochain** [comment]
@gamalielhere I can see that there is communication going through, but sometimes is fails to reuse the opened websocket (sometimes the socket is already closed). Anything I could do from my end to help you?

image

**Comment by r-gochain** [comment]
@gamalielhere please test once again, I've submitted a couple more fixes, now it shouldn't occasionally break the connection

**Comment by gamalielhere** [comment]
@r-gochain just got done testing it, the new changes seems to have fixed the issue we're having. Thanks for looking into it and fixing it!

**Pull request closed without merging**